### ISSUE TITLE
Feature: PI: display Point estimate and CI intervals in two columns

### ISSUE THREAD

**Issue by StephanSchaller** [Open-Systems-Pharmacology/PK-Sim#1580]
It would be convenient if the Point estimate and CI intervals are displayed in two columns and not as one as is currently done:
![image](https://user-images.githubusercontent.com/25477643/97440405-b40b7200-1927-11eb-838e-9792f304e04d.png)

Also, it might be more convenient to express the CI interval in % of the point estimate to make it easier to quantify on sight (or allow switching from Parameter Unit to %)

**Comment by msevestre** [comment]
Agrred on both points

**Comment by hainiwen** [comment]
hey Stephan! Does PKSIM or Mobi still do that? 
From my screen, I don't think I can get 95% CI of my optimized parameters. 
I happen to be reading this paper https://bpspubs.onlinelibrary.wiley.com/doi/10.1111/bcp.12234
It mentioned that many papers ignored 95% CI but now I realize I don't know where it is computed

**Comment by StephanSchaller** [comment]
Yes, make sure you have selected the box "calculate sensitivities", which will then also calculate the CI.
Let me know if you have troubles.

Stephan
